db/migrate/20200114204550_add_more_commentary_to_uploads.rb

```diff
@@ -0,0 +1,10 @@
+class AddMoreCommentaryToUploads < ActiveRecord::Migration[6.0]
+  def change
+    add_column :uploads, :translated_commentary_title, :text
+    add_column :uploads, :translated_commentary_desc, :text
+    add_column :uploads, :add_commentary_tag, :boolean
+    add_column :uploads, :add_commentary_request_tag, :boolean
+    add_column :uploads, :add_commentary_check_tag, :boolean
+    add_column :uploads, :add_partial_commentary_tag, :boolean
```

Comment:
These should all have `null: false` and defaults set (`default: false` for booleans and `default: ""` for strings`).